lisa/sut_orchestrator/azure/features.py

```diff
@@ -1665,6 +1691,11 @@ def get_hardware_disk_controller_type(self) -> Any:
         vm = get_vm(azure_platform, self._node)
         return vm.storage_profile.disk_controller_type
 
+    def get_ephemeral_disk_placement_type(self) -> Any:
```

Comment:
The method name is not accurate, it should mention it's for os disk.

Reply:
> The method name is not accurate, it should mention it's for os disk.

Hmm actually in Azure, it looks like disk placement is configured only for Ephemeral disk type. 
See here - https://learn.microsoft.com/en-us/powershell/module/az.compute/set-azvmssstorageprofile?view=azps-13.0.0
But if you think, the functional name should be generic, I can change it. let me know your preference after reading through above link.

Reply:
But the L1697 called for `os_disk`.
```vm.storage_profile.**os_disk**.diff_disk_settings.placement```